pom.xml

```diff
@@ -141,12 +130,11 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.8.1</version>
+                    <version>${maven-compiler-plugin.version}</version>
                     <configuration>
+                        <release>14</release>
```

Comment:
You still need this, in addition to the java.version above?

Reply:
Ah! Was acttually meant to use ${java.version} in there!!! It is needed so that children of the parent also get this set. Just stating <java.version> in your properties section will not be inherited so child projects wouldn't be using the correct java version. Specifying the plugin config in teh pluginManagement section is teh way to get this to child projects.